### ISSUE TITLE
[BUG] The app is only visible when running as an administrator.

### ISSUE THREAD

**Issue by hanguk0726** [YehudaKremer/msix#204]
### :information_source: Info

msix: ^3.14.1

### :speech_balloon: Description

The app is only visible when running as an administrator.
I downloaded it for testing from the Microsoft store, but I can't see the app. I checked that it is in the Taskmanager.
When run as an administrator, it works fine.

### :scroll: Pubspec.yaml
environment:
  sdk: '>=3.0.2 <4.0.0'

msix_config:
  display_name: 
  publisher_display_name:   (values from ms partner center)
  identity_name: 
  publisher: 
  msix_version: 1.0.0.0
  logo_path: 
  store: true
  capabilities: microphone, webcam

**Comment by hanguk0726** [comment]
I changed win32_window.cpp as described in the link.
The window now appears, but the app function still works normally only with administrator privilege.
Is there anything in the msix setup that is related to the administrator privilege I missed?

**Comment by YehudaKremer** [comment]
What package ("dependencies") you using?

**Comment by hanguk0726** [comment]
Here is the list.
dependencies:
  flutter:
    sdk: flutter

  cupertino_icons: ^1.0.2
  irondash_message_channel: ^0.1.0
  irondash_engine_context: ^0.1.0
  flutter_spinkit: ^5.1.0
  provider: ^6.0.5
  rxdart: ^0.27.7
  window_manager: ^0.3.2
  path: ^1.8.3
  path_provider: ^2.0.14
  objectbox: ^2.0.0
  objectbox_flutter_libs: any
  media_kit: ^0.0.5                            
  media_kit_video: ^0.0.7                     
  media_kit_native_event_loop: ^1.0.3            
  media_kit_libs_windows_video: ^1.0.2         
  fullscreen_window: ^1.0.4
  slider_controller: ^0.0.5
  audio_video_progress_bar: ^1.0.1
  intl: ^0.18.1
  wakelock: ^0.6.2
  easy_debounce: ^2.0.3
  sentry_flutter: ^7.5.2
  package_info_plus: ^3.1.2
  
dev_dependencies:
  flutter_test:
    sdk: flutter

  flutter_lints: ^2.0.0
  build_runner: ^2.4.1
  objectbox_generator: any

**Comment by YehudaKremer** [comment]
Sorry for the response delay,
Msix installation is not adding requirement for administrator privileges.

When you run your app without msix installation ("after running `flutter build windows`"), can run the .exe file without administrator privileges?

**Comment by hanguk0726** [comment]
Yes, after 'flutter build windows' without msix, exe files work normally without administrator privileges. I think there is a chance that this issue is on the Microsoft Store.
